industry/products.md

```diff
@@ -26,38 +26,52 @@ Raw models usually do not include the features included in many production syste
 - HTML/XML tag handling
 
 
-| Company | Product | Self-serve |
+| Product | Company | Self-serve |
 | --- | --- | :-: |
-| Google | Google Translate | &#10004; |
-| [LingvaNex](/../industry/companies.md#lingvanex) | Translation API | &#10004; |
-| Yandex | Translate API | &#10004; |
-| Microsoft | Translator Text API | &#10004; |
-| Amazon | Translate API | &#10004; |
-| [ModernMT](/../industry/companies.md#modernmt) | API | &#10004; |
-| IBM | Watson Language Translator | &#10004; |
-| Baidu | 翻译API | &#10004; |
-| [DeepL](/../industry/companies.md#deepl) | API | &#10004; |
-| Alibaba | Machine Translation | &#10004; |
-| [Systran](/../industry/companies.md#systran) | Systran Translate Pro | &#10004; |
-| [Unbabel](/../industry/companies.md#unbabel) | Unbabel Translate | |
-| [Lilt](/../industry/companies.md#lilt) | Lilt Translate | |
-| [Tilde](/../industry/companies.md#tilde) | TildeMT | |
-| [Omniscien](/../industry/companies.md#omniscien-technologies) | Language Studio | |
-| [KantanMT](/../industry/companies.md#kantanmt) | Translate AI | |
-| [Pangeanic](/../industry/companies.md#pangeanic) | PangeaMT | |
-| TextShuttle | TextShuttle AI Translation | |
-| Globalese | Globalese | |
-| RWS Group | Language Weaver (Iconic) | &#10004; |
-| Mirai Translate | Mirai Translator | &#10004; |
+| Google Translate | Google | &#10004; |
+| Translation API | [Lingvanex](/../industry/companies.md#lingvanex) | &#10004; |
+| Translate API | Yandex | &#10004; |
+| Translator Text API | Microsoft | &#10004; |
+| Translate API | Amazon | &#10004; |
+| ModernMT API | [Translated](/../industry/companies.md#translated) | |
+| Watson Language Translator | IBM | &#10004; |
+| 翻译API | Baidu | &#10004; |
+| API | [DeepL](/../industry/companies.md#deepl) | &#10004; |
+| Machine Translation | Alibaba | &#10004; |
+| Systran Translate Pro | [SYSTRAN](/../industry/companies.md#systran) | &#10004; |
+| Unbabel Translate | [Unbabel](/../industry/companies.md#unbabel) | |
+| Lilt Translate | [Lilt](/../industry/companies.md#lilt) | |
+| TildeMT | [Tilde](/../industry/companies.md#tilde) | |
+| Language Studio | [Omniscien Technologies](/../industry/companies.md#omniscien-technologies) | |
+| Translate AI | [KantanMT](/../industry/companies.md#kantanmt) | |
+| PangeaMT | [Pangeanic](/../industry/companies.md#pangeanic) | |
+| TextShuttle AI Translation | [TextShuttle](/../industry/companies.md#textshuttle) | |
+| Globalese API | Globalese | |
+| API | [ImageTranslate](/../industry/companies.md#imagetranslate) | |
```

Comment:
"API" probably isn't the best name

Reply:
Maybe we just have one column?

And include the company name if it's not part of the name

And bold the company name and hyperlink the company name if we have it

e.g.

> | | Self-serve |
> | -- | -- |
> | **Microsoft** Translator API | ✔ |
> | **Amazon** Translate API | ✔ |
> | [ModernMT](/../industry/companies.md#modernmt) API | ✔ |
> | **IBM** Watson Language Translator  |  ✔ |

I was actually wrong about ModernMT, ModernMT was its own company (until it was acquired yesterday).  Translated was just an investor.  So we should update the companies page to reflect that.

Reply:
> I was actually wrong about ModernMT, ModernMT was its own company (until it was acquired yesterday). Translated was just an investor. So we should update the companies page to reflect that.

Do you mean it was acquired not by Translated but by another company? I've read that it was merged by Translated, which acquired all stocks in ModernMT. So I've updated the info both in the `Companies` and `Products` pages.

Reply:
It was acquired by Translated.

Reply:
I guess we can remove Translated from companies?  It's just an LSP.